tracer/test/Datadog.Trace.Tests/Telemetry/config_prefix_block_list.json

```diff
@@ -70,6 +70,8 @@
   "DD_Kafka_",
   "DD_TRACE_CosmosDb_",
   "DD_CosmosDb_",
+  "DD_TRACE_AwsLambda_",
+  "DD_AwsLambda_",
```

Comment:
Just FYI, don't forget to do a PR to the telemetry repo to mirror this change, thanks! 🙂

Reply:
This is added in this PR: https://github.com/DataDog/dd-go/pull/106971